### PR TITLE
DFD-164 redesign tutorial

### PR DESCRIPTION
**Pull request by 0xhank**
Description: Tutorial and help pane redesign

To test:
- [ ] Walk through each step of the tutorial checking for grammar/styling issues
- [ ] Ensure that all panes make logical sense and the content isn't confusing
- [ ] Review the help pane for grammar/content and that general flow makes sense

### CODE REVIEW

File: src/Frontend/Utils/constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { address } from '@darkforest_eth/serde';
-import { BadgeType, EthAddress, GrandPrixMetadata, LiveMatch } from '@darkforest_eth/types';
+  WorldCoords,
```
Comment (lermchair) [inline src/Frontend/Utils/constants.ts]: was missing a bunch of imports

**Review by lermchair — APPROVED** [review]
🐑

**Comment by cha0sg0d** [comment]
Playing tutorial rn. Feedback:

1. Make sure text says `move from spawn planet to asteroid`
2. Make sure enough energy is sent to an Asteroid to immediately move silver back (>50%(
3. `Hint:before moving` needs space m
4. Map: Spaceship should be a gear and should only go to a Foundry. That Foundry should be quite close to the spawn planet
5. You can pause your explorer when the map is fully revealed (up to the white circle)
6. Add ability to restart tutorial and go back to the beginning.
Screen Shot 2022-09-07 at 10 16 45 AM
7. This is confusing when first loading to do tutorial.
8. Does tutorial only work the first time a player joins? I tried to do it again and got the normal Grand Prix welcome.

**Comment by 0xhank** [comment]
4. I think im going to change the foundry to a mothership or something a bit simpler
5. what does this mean
6. This feels like a low priority feature we can add down the line
7. This doesn't load for me in the tutorial world.

**Comment by cha0sg0d** [comment]
More thoughts:
1. Clicking on my spaceship instantly moves to selecting the Target Planet, even tho I haven't moved my spaceship.
2. re 5, I think you should add text that says `you can pause the explorer when all of the space within the white circle is revealed` in this pane 
Screen Shot 2022-09-07 at 12 18 43 PM
3. I think finding and prospecting an Artifact is v important for understanding the game, so I think gear is good. just add an admin foundry nearby

**Review by cha0sg0d — APPROVED** [review]
Good First Draft, we will iterate further

**Pull request merged**